registration-processor/workflow-engine/registration-processor-workflow-manager-service/src/main/java/io/mosip/registration/processor/workflowmanager/WorkflowManagerApplication.java

```diff
@@ -33,7 +34,9 @@ public static void main(String[] args) {
 		workflowInternalActionVerticle.deployVerticle();
 		WorkflowActionApi workflowActionApi = ctx.getBean(WorkflowActionApi.class);
 		workflowActionApi.deployVerticle();
-		WorkflowActionJob workflowActionJob = ctx.getBean(WorkflowActionJob.class);
-		workflowActionJob.deployVerticle();
+//		WorkflowActionJob workflowActionJob = ctx.getBean(WorkflowActionJob.class);

```

Comment:
Please uncomment this 37 and 38 lines to execute jobs

Reply:
done ,please check earliest commit